.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+    root: true,
+    parser: '@typescript-eslint/parser',
+    parserOptions: {
+        tsconfigRootDir: __dirname,
+        project: ['./tsconfig.json'],
+    },
+    plugins: [
+        '@typescript-eslint',
+    ],
+    extends: [
+        'eslint:recommended',
+        'plugin:@typescript-eslint/recommended',
```

Comment:
for react there is a plugin that checks that the rules of hooks are followed ( "plugin:react-hooks/recommended"). Can we use that as well?

Reply:
Nice one. Done.